lib/rack/session/redic.rb

```diff
@@ -48,7 +48,7 @@ def initialize(app, options = HASH)
 
         @expires = options[:expire_after]
         @marshaller = options.fetch(:marshaller, Marshal)
-        @storage = ::Redic.new(options.fetch(:url, ENV.fetch(REDIS_URL)))
+        @storage = ::Redic.new(options.fetch(:url) { |k| ENV.fetch(REDIS_URL) })
```

Comment:
Since it's unused I believe you can leave the block argument off.

```suggestion
        @storage = ::Redic.new(options.fetch(:url) { ENV.fetch(REDIS_URL) })
```